ampersand-checkbox-view.js

```diff
@@ -1,21 +1,18 @@
 /*$AMPERSAND_VERSION*/
 var View = require('ampersand-view');
-var domify = require('domify');
 var dom = require('ampersand-dom');
 
 
-// can be overwritten by anything with:
-// <input>, <label> and the same `data-hook` attributes
-var template = [
-    '<label class="checkbox">',
-        '<input type="checkbox"><span data-hook="label"></span>',
-        '<div data-hook="message-container" class="message message-below message-error">',
-            '<p data-hook="message-text"></p>',
-        '</div>',
-    '</label>'
-].join('');
-
 module.exports = View.extend({
+    template : [
+        '<label class="checkbox">',
+            '<input type="checkbox"><span data-hook="label"></span>',
+            '<div data-hook="message-container" class="message message-below message-error">',
+                '<p data-hook="message-text"></p>',
+            '</div>',
+        '</label>'
+        ].join(''),
```

Comment:
the js engine may optimize for this, but i think this was left out to reduce per-instance mem/copies, vs. a single, static ref.  i'd be interested to know if anyone has done some testing on this memory perf case